pkg/flare/archive_linux.go

```diff
@@ -20,14 +25,72 @@ func zipLinuxKernelSymbols(tempDir, hostname string) error {
 	return zipLinuxFile("/proc", tempDir, hostname, "kallsyms")
 }
 
-func zipLinuxKrobeEvents(tempDir, hostname string) error {
+func zipLinuxKprobeEvents(tempDir, hostname string) error {
 	return zipLinuxFile("/sys/kernel/debug/tracing", tempDir, hostname, "kprobe_events")
 }
 
 func zipLinuxPid1MountInfo(tempDir, hostname string) error {
 	return zipLinuxFile("/proc/1", tempDir, hostname, "mountinfo")
 }
 
+var klogRegexp = regexp.MustCompile(`<(\d+)>(.*)`)
+
+func readAllDmesg() ([]byte, error) {
```

Comment:
Isn't this forbidden by Linux if not the `root` user?

Reply:
According to the klogctl manpage:
`All commands except SYSLOG_ACTION_READ_ALL and SYSLOG_ACTION_SIZE_BUFFER require privilege.`. For safety, I don't fail - only log - when there is an error